public/externalLibs/streams/stream.js

```diff
@@ -95,13 +94,12 @@ function stream_length(xs) {
 // first argument is not a function.
 // Lazy? Yes: The argument stream is only explored as forced by
 //            the result stream.
-function stream_map(f, s) {
-  if (is_null(s)) {
+function stream_map(f: Function, s: Stream): Stream | null {
```

Comment:
There shouldn't be any Typescript declarations here

Reply:
Oh, thanks for catching this.